multi-node/aws/pkg/config/config.go

```diff
@@ -141,6 +147,15 @@ func (c Cluster) Config() (*Config, error) {
 	return &config, nil
 }
 
+func (c Cluster) CustomEtcdEndpoint() bool {
```

Comment:
I would call this function `EtcdOnController` and then `return host == c.ControllerIP`.